QUARKUS-3461.md

```diff
@@ -0,0 +1,41 @@
+# QUARKUS-3461 Verify behaviour and functionality of Quarkus CLI update command with RHBQ
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3461
+- Quarkus guides:
+    - https://quarkus.io/guides/update-quarkus
+
+## Scope of the testing
+- Exploratory testing of the feature over supported versions of RHBQ and Quarkus upstream
+- Writing test coverage for this feature is continual work, requires update for every release of Quarkus
+- Test update between each of Quarkus LTS release versions
+- Test update of multi-module projects
```

Comment:
Quick check, is this even possible right now? I remember trying it on TS when javax -> jakarta was required and it didn't work. I cannot see it https://quarkus.io/guides/update-quarkus. It's ok as long as it is expected to work.

Reply:
I think it's not possible now, we need to push dev to support this feature. I think, test update of complex RHBQ applications is necessary. Last time we talk with @rsvoboda, he mentioned that we need test complex applications

Reply:
thanks, let's keep it then